### PR TITLE
general fixes

### PR DESCRIPTION
**Pull request by chetanyagoyal**
1. Fixes the opamp spice testbench simulations, noticed it was failing in the CI
2. Fixes some general issues in code import in the relational framework
  - bolsters error logging of the import statements using try, except blocks

### CODE REVIEW

File: openfasoc/generators/glayout/glayout/flow/pdk/mappedpdk.py
Comment (alibillalhammoud) [inline openfasoc/generators/glayout/glayout/flow/pdk/mappedpdk.py:1]: remove git keeps

File: openfasoc/generators/glayout/glayout/syntaxer/relational.py
Comment (alibillalhammoud) [inline openfasoc/generators/glayout/glayout/syntaxer/relational.py:1]: Do you want me to add my changes for this, I can commit them to your fork

**Review by alibillalhammoud — CHANGES_REQUESTED** [review]
make changes as described, otherwise LGTM

File: openfasoc/generators/glayout/glayout/syntaxer/relational.py
Comment (chetanyagoyal) [reply openfasoc/generators/glayout/glayout/syntaxer/relational.py:1]: yes sounds good

**Review by msaligane — APPROVED** [review]

**Pull request merged**